### DISCUSSION TITLE
Question regarding the FOUND images

### DISCUSSION THREAD

**Discussion by hajo62** [RouHim/this-week-in-past#164] [Announcements]
The endpoint .../api/resources/week/metadata shall show the metadata of **all** matching images?

If so...
319 images are found.
I have a folder containing 41 images all taken at the same date (1996-09-14). 
The metadata endpoint shows just 1 of this images.

Shouldn't there be all 41 images listed?

**Comment by hajo62** [comment]
Checked one day later. Still just 1 image in the list. 
The 40 other images are valid images and have the same credentials. 
Any ideas what to check or how to dig in?

**Comment by RouHim** [comment]
This is a debug endpoint which:
Gets a list of all visible resources for the current "week"

**Comment by hajo62** [comment]
> Gets a list of all visible resources for the current "week"

Than it should list all items with same date, I assume?
But I see only one of those in the list.

**Comment by hajo62** [comment]
To narrow down the dataset to reproduce.
If I start my second instance with it still the same.
```
      RESOURCE_PATHS: |-
        /resources/1990-1999/1996
```

**Comment by hajo62** [comment]
Regarding https://github.com/RouHim/this-week-in-past/issues/138#issuecomment-2348859419:
I narrowed down to 3 images in two folders, but haven't found an email to send to...

I assume it's some strange content in one of the those images which causes the app to fail.

**Comment by RouHim** [comment]
Thank you very much, feel free to send me your examples:
rouven@himmelstein.info

**Comment by hajo62** [comment]
> send me your examples

Done.

**Comment by RouHim** [comment]
Thank you. I'll get back to you, if I found something.

**Comment by hajo62** [comment]
You moved this to discussion; is more information needed, why the find images algo does not return all images?
